packages/wallets/src/components/AppCard/AppCard.tsx

```diff
@@ -25,10 +26,11 @@ const AppCard: React.FC<TProps> = ({
     cardSize = 'md',
     device = 'desktop',
     isDemoWallet = false,
+    testId,
     walletName,
 }) => {
     return (
-        <div className={`wallets-app-card wallets-app-card--border-radius--${cardSize}`}>
+        <div className={`wallets-app-card wallets-app-card--border-radius--${cardSize}`} data-testid={testId}>
```

Comment:
the general convention so far was that components were responsible for handling their own data-testid by themselves, not via props. Any reason to change that? 
Feels better to stick to convention and have consistency across codebase.

Reply:
I wasn't aware of this, I'll proceed to remove the test Ids here